ui/nuxeo-user-group-management/nuxeo-create-user.js

```diff
@@ -212,8 +212,22 @@ import '../nuxeo-button-styles.js';
             this._goHome();
           }
         })
-        .catch((error) => {
-          this.errors = error.message;
+        .catch(async (error) => {
+          let message = error;
+          if (error.message) {
+            ({ message } = error);
```

Comment:
`()`? 🤔

Reply:
This is to make the linter happy, AFAIK.

Reply:
if you start with curly braces you are starting a block statement, it's more explicit if you wrap those around parenthesis